app/controllers/grant_permissions_controller.rb

```diff
@@ -1,18 +1,15 @@
 class GrantPermissionsController < ApplicationController
-  include WithGrantRoles
-  include UsersHelper
+  # include UsersHelper
```

Comment:
Should this come out totally, if it is not being used? or is there a reason to keep the comment?

Reply:
Interesting catch. Would have caused errors but since the UsersHelper is included in the system spec, the controller had access to the needed method.